app.json

```diff
@@ -513,6 +524,69 @@
           }
         }
       ]
+    },
+    {
+      "id": "growatt",
+      "name": {
+        "en": "Growatt",
+        "nl": "Growatt"
+      },
+      "pair": [
+        {
+          "id": "login_credentials",
+          "template": "login_credentials",
+          "options": {
+            "logo": "pair.svg"
+          },
+          "navigation": {
+            "next": "list_devices"

```

Comment:
Do not allow a user to click next here

Reply:
Fixed (and tested OK: App does navigate to device selection after username & password have been entered).